infra/terraform/external_function/snowalert/lambda-code/drivers/process_https.py

```diff
@@ -50,7 +51,7 @@ def process_row(
     }
     req_headers.setdefault('User-Agent', 'Snowflake SnowAlert External Function 1.0')
     if auth:
-        auth = decrypt_if_encrypted(auth)
+        auth = str(decrypt_if_encrypted(auth))
```

Comment:
this shouldn't be necessary -- looking at [the docs](https://mypy.readthedocs.io/en/stable/common_issues.html) maybe mypy needs an explicit

    assert auth is not None

here?